steps/actions/elasticsearch-run.dhall

```diff
@@ -0,0 +1,19 @@
+let Step = ../../schemas/Step.dhall
+
+in      \(args : { stack-version : Text, nodes : Optional Natural })
+    ->  let nodes-vars =
+              merge
+                { None = [] : List { mapKey : Text, mapValue : Text }
+                , Some =
+                    \(nodes : Natural) -> toMap { nodes = Natural/show nodes }
+                }
+                args.nodes
+
+        in  Step::{
+            , id = None Text
+            , name = Some "Runs elasticsearch version ${args.stack-version}"
+            , uses = Some "elastic/elastic-github-actions/elasticsearch@master"
```

Comment:
@TristanCacqueray It's fine to add it that's what `steps` package is for. Can you just move this to `steps/elastic`? I'm namespacing steps according to `org/repo`